server/controllers/userController.js

```diff
@@ -61,8 +61,32 @@ userController.verifyUser = async (req, res, next) => {
   }
 };
 
+// this is going to be coming from a patchrequest to add the board we juts created into the usersBoard that created it
+  // the request body I'll receive is {username: username}
+  // find user by username and update === findOneAndUpdate method thru mongoose
+
+  userController.addBoardId = (req, res, next) => {
+  const { username } = req.body;
+  const board_id = res.locals.board._id;
+  User.findOneAndUpdate({username: username},  
+    {$push: { board_ids: board_id }}, 
+    {new: true}).exec()
+      .then(data => {
+        res.locals.user = data; //not using this at the moment
```

Comment:
Do we have future plans for using this? If so what would it be?

Reply:
I don't think we had any plans for it, this was just the result using findOneAndUpdate and we decided to leave it there.

Reply:
No future plans. Can be removed with no loss of function.

Reply:
Am I supposed to delete the code through here or push it again with code removed?